src/Sticky.js

```diff
@@ -40,41 +42,60 @@ export default class Sticky extends Component {
   }
 
   handleContainerEvent = ({ distanceFromTop, distanceFromBottom, eventSource }) => {
-    const parent = this.context.getParent();
-    const preventingStickyStateChanges = this.props.relative && eventSource !== parent;
+  const parent = this.context.getParent();
+  const preventingStickyStateChanges = this.props.relative && eventSource !== parent;
 
-    const placeholderClientRect = this.placeholder.getBoundingClientRect();
-    const contentClientRect = this.content.getBoundingClientRect();
-    const calculatedHeight = contentClientRect.height;
+  const placeholderClientRect = this.placeholder.getBoundingClientRect();
+  const contentClientRect = this.content.getBoundingClientRect();
+  const calculatedHeight = contentClientRect.height;
 
-    const bottomDifference = distanceFromBottom - this.props.bottomOffset - calculatedHeight;
+  const bottomDifference = distanceFromBottom - this.props.bottomOffset - calculatedHeight;
 
-    distanceFromTop = -distanceFromTop + this.placeholder.offsetTop;
+  distanceFromTop = -distanceFromTop + this.placeholder.offsetTop;
 
-    const wasSticky = !!this.state.isSticky;
-    const isSticky = preventingStickyStateChanges ? wasSticky : (distanceFromTop < -this.props.topOffset && distanceFromBottom > -this.props.bottomOffset);
+  const wasSticky = !!this.state.isSticky;
 
-    distanceFromBottom = (this.props.relative ? parent.scrollHeight - parent.scrollTop : distanceFromBottom) - calculatedHeight;
+  distanceFromBottom = (this.props.relative ? parent.scrollHeight - parent.scrollTop : distanceFromBottom) - calculatedHeight;
 
+  let isSticky = false;
+  let topValue = 0;
 
-    const style = !isSticky ? { } : {
-      position: 'fixed',
-      top: bottomDifference > 0 ? (this.props.relative ? parent.offsetTop - parent.offsetParent.scrollTop : 0) : bottomDifference,
-      left: placeholderClientRect.left,
-      width: placeholderClientRect.width,
-      transform: this.props.disableHardwareAcceleration ? '' : 'translateZ(0)'
-    }
+  if (!this.props.isFooter) {
```

Comment:
hmmm definitely doesnt work for relative yet.  Will make changes for this as well